content/400-pulse/400-api-reference.mdx

```diff
@@ -1,15 +1,13 @@
 ---
-title: 'API reference'
```

Comment:
I think you ran prettier (or something similar) on this file. Could we remove those changes? (sorry)

Reply:
Just for my own understanding: What's the issue with that? Just consistency with other files?

Reply:
I generally think double-quotes are preferable in frontmatter because they allow usage of single quotes in the frontmatter content